docs/jwst/cube_build/main.rst

```diff
@@ -57,6 +57,20 @@ a single NIRSpec SCA. In high resolution mode the 30 slices are imaged on the tw
 Terminology
 -----------
 
+General IFU Terminology
++++++++++++++++++++
+
+``pixel``
+  A pixel is a physical 2-D element of the detector focal plane arrays.
+
+``spaxel``
+  A spaxel is a 2-D spatial element of an IFU rectified data cube.  Each spaxel in a data cube
+  has an associated spectrum composed of many voxels.
```

Comment:
Sorry, but if a spaxel is 2D and a voxel is 3D, I would assume that a voxel is composed of a string of spaxels, where each spaxel contains data at a different wavelength. So how then can a spaxel be composed of many voxels? Seems to me like it would be the other way around. But then I'm not all that familiar with IFU lingo.

Reply:
Thinking of a data cube like a city full of skyscrapers (wavelength in the vertical direction), ’spaxel’ describes a particular city block, while ‘voxel’ describes a specific floor on that block.